tests/e2e/instrumentation-apache-httpd/01-install-app.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: my-apache
+spec:
+  selector:
+    matchLabels:
+      app: my-apache
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: my-apache
+      annotations:
+        sidecar.opentelemetry.io/inject: "true"
+        instrumentation.opentelemetry.io/inject-apache-httpd: "true"
+    spec:
+      containers:
+      - name: myapp
+        image: docker.io/chrlic/apache-test
```

Comment:
What's the origin of this image? what's the license on it?

Reply:
image origin and license info included in comments.
also added image checksum